addon/components/sortable-group.js

```diff
@@ -111,7 +118,11 @@ export default Component.extend({
         items.invoke('thaw');
       });
     });
-
-    this.sendAction('onChange', models);
+    
+    if (groupModel === null) {
```

Comment:
I’m not sure we can compare with `null` because it’s possible the object passed as `model` could be `null` too.

Let’s say the rule is:

If you write `{{#sortable-group model=anything ...}}` in your template then you get the two argument action, no matter what `anything` evaluates to.